api/swagger.yml

```diff
@@ -556,6 +556,17 @@ components:
         email:
           type: string
 
+    InviteUserRequest:
+      type: object
+      required:
+        - email
+        - id
```

Comment:
Why does an invite user request require an id? how would the user have an id at this point?

Reply:
The id here is just the name of the user (Just like when creating a user, see auth/user/). The idea here is to preserve backend compatibility for createUser as much as possible.